sal_interfaces/SALSubsystems.xml

```diff
@@ -2,6 +2,29 @@
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALSubsystemSet.xsl"?>
 <SALSubsystemSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALSubsystemSet.xsd">
+<SALSubsystem>
+  <Name>AdamSensors</Name>
+  <Description>CSC ADAM 6024 device connected to up to 6 transducer type sensors</Description>
+  <Enumeration></Enumeration>
+  <Generics>yes</Generics>
+  <Version>0.0.0</Version>
+  <Author></Author>
+  <ActiveDevelopers>Colin Winslow</ActiveDevelopers>
+  <PrincipalCSCOwner>Patrick Ingraham</PrincipalCSCOwner>
+  <Github>https://github.com/lsst-ts/ts_adamSensors</Github>
+  <JenkinsTestResults>NA</JenkinsTestResults>
+  <LSSTPoC>Patrick Ingriham</LSSTPoC>
+  <CSCDocs>NA</CSCDocs>
+  <ProductOwner>Patrick Ingraham</ProductOwner>
+  <RelatedDocuments></RelatedDocuments>
+  <SoftwareLanguage>Python</SoftwareLanguage>
+  <RuntimeLanguages>CPP,LabVIEW,Java,PyDDS,Python</RuntimeLanguages>
```

Comment:
Admittedly, I don't know what this is... but i suppose i'm a little surprised to see Labview/Java/CPP here ??

Reply:
I actually just left this one the same as the subsystem i copied/pasted as a template, simply because i'm not sure what the scope of what gets included as a runtime language should be. For example, a bunch of numpy is actually compiled C++ with python wrappers around it. But everything I'm writing is just python.

Reply:
Leaving it as-is for now, to be possibly revisited at a future date